frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelVersion/modelVersionDetails.cy.ts

```diff
@@ -271,19 +276,28 @@ describe('Model version details', () => {
     });
 
     it('Model version details registered from catalog', () => {
-      initIntercepts(false, true);
+      initIntercepts(false, true, true);
       modelVersionDetails.visit();
       modelVersionDetails.findVersionId().contains('1');
       modelVersionDetails.findRegisteredFromCatalog().should('exist');
       modelVersionDetails
         .findRegisteredFromCatalog()
-        .should('have.text', 'test-catalog-model (test-catalog-tag) in Model catalog');
+        .should('have.text', 'test-catalog-model (test-catalog-tag)');
       modelVersionDetails.findRegisteredFromCatalog().click();
       verifyRelativeURL(
         '/modelCatalog/test-catalog-source/test-catalog-repo/test-catalog-model/test-catalog-tag',
       );
     });
 
+    it('Model version details registered from catalog with model catalog unavailable', () => {
+      initIntercepts(false, true, false);
+      modelVersionDetails.visit();
+      modelVersionDetails.findVersionId().contains('1');
+      cy.contains('test-catalog-model (test-catalog-tag) in Model catalog').should('exist');
+      modelVersionDetails.findRegisteredFromCatalog().should('not.exist');
+      cy.contains('Registered from').should('exist');
```

Comment:
same goes for this

Reply:
I think what @manaswinidas meant here is to use the testID just for cy.contains(...).